tests/python_client/testcases/test_utility.py

```diff
@@ -208,10 +209,13 @@ def test_loading_progress_invalid_collection_name(self, invalid_c_name):
         self._connect()
         c_name = cf.gen_unique_str(prefix)
         df = cf.gen_default_dataframe_data()
-        self.collection_wrap.construct_from_dataframe(c_name, df, primary_field=ct.default_int64_field_name)
+        self.collection_wrap.construct_from_dataframe(
+            c_name, df, primary_field=ct.default_int64_field_name)
         self.collection_wrap.load()
-        error = {ct.err_code: 1, ct.err_msg: "Invalid collection name: {}".format(invalid_c_name)}
-        self.utility_wrap.loading_progress(invalid_c_name, check_task=CheckTasks.err_res, check_items=error)
+        error = {ct.err_code: 1, ct.err_msg: "Invalid collection name: {}".format(
+            invalid_c_name)}
+        self.utility_wrap.loading_progress(
+            invalid_c_name, check_task=CheckTasks.err_res, check_items=error)
```

Comment:
@yah01 Is this the auto formatting of IDE? If so, please turn this auto formatting off and redo these changes, those line-breaking to limit  the length of  one line is awful and inconsistent with other files